src/plugins/intel_gna/descriptions/gna_desc.hpp

```diff
@@ -0,0 +1,151 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include <numeric>
+#include <functional>
+#include <unordered_set>
+
+#include "ie_precision.hpp"
+#include "ie_input_info.hpp"
+#include "ie_algorithm.hpp"
+
+#include "backend/dnn_types.h"
+#include "gna_plugin_config.hpp"
+
+namespace GNAPluginNS {
+
+/*
+ * This base structure accumulates all required information for network inputs and outputs
+ */
+struct GnaDesc {
+    // common OV properties
+    std::string name;
+    std::unordered_set<std::string> tensor_names;
+    InferenceEngine::Layout layout;
+    InferenceEngine::SizeVector dims;
+    InferenceEngine::Precision precision;
+    InferenceEngine::Precision blob_precision;
```

Comment:
this is not the tensor precision exactly, because we are using this filed to define the precision after quantization, but this name looks better, changed it

Reply:
Changed